grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -902,6 +902,10 @@ def section_streaming_actors(outer_panels):
                             f"rate({metric('stream_join_insert_cache_miss_count')}[$__rate_interval])",
                             "cache miss when insert {{side}} side, join_table_id {{join_table_id}} degree_table_id {{degree_table_id}} actor {{actor_id}}",
                         ),
+                        panels.target(
+                            f"rate({metric('stream_temporal_join_cache_miss_count')}[$__rate_interval])",
+                            "temporal join cache miss, table_id {{table_id}} actor {{actor_id}}",
+                        ),
```

Comment:
temporal join does not use the hash join executor.  Maybe rename the `Join Executor Cache` to  `Hash Join Executor Cache` and add a `Temporal Join Executor Cache`